CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # CHANGELOG
 
+## 1.30.0 (2022-10-19)
+
+MLflow 1.30.0 includes several major features and improvements
+
+Features:
+
+- [Pipelines] Introduce hyperparameter tuning support to MLflow Pipelines (#6859, @prithvikannan)
+- [Pipelines] Add support for AutoML in MLflow Pipelines using FLAML (#6959, @mshtelma)
+- [Tracking] Introduce `run_name` attribute for `create_run`, `get_run` and `update_run` APIs (#6782, #6798 @apurva-koti)
+- [Tracking] Add support for searching by `creation_time` and `last_update_time` for the `search_experiments` API  (#6979, @harupy)
+- [Tracking] Add support for search terms `run_id IN` and `run ID NOT IN` for the `search_runs` API (#6945, @harupy)
+- [Tracking] Add support for searching by `user_id` and `end_time` for the `search_runs` API (#6881, #6880 @subramaniam02)
+- [Tracking] Add support for searching by `run_name` and `run_id` for the `search_runs` API (#6899, @harupy; #6952, @alexacole)
+- [Tracking] Add support for synchronizing run `name` and run name tag (#6971, @BenWilson2)
+- [Tracking] Add support for signed tracking server requests using AWSSigv4 and AWS IAM (#7044, @pdifranc)
+- [Tracking] Introduce the `update_run()` API for modifying the `status` and `name` attributes of existing runs (#7013, @gabrielfu)
+- [Tracking] Add support for experiment deletion in the `mlflow gc` cli API (#6977, @shaikmoeed)
+- [Models] Add support for environment restoration in the `evaluate()` API (#6728, @jerrylian-db)
+- [Models] Remove restrictions on binary classification labels in the `evaluate` API (#7077, @dbczumar)
+- [Scoring] Add support for `BooleanType` to `pyfunc` spark udf (#6913, @BenWilson2)
+- [Sqlalchemy] Add support for configurable `Pool` class options for `sqlalchemy` (#6883, @mingyu89)
+
+Bug fixes:
+
+- [Pipelines] Enable Pipeline subprocess commands to create a new `SparkSession` if one does not exist (#6846, @prithvikannan)
+- [Pipelines] Fix an issue in rendering `bool` column types for `pandas_renderer()` (#6907, @sunishsheth2009)
+- [Tracking] Fix a bug in the handling of run names defined as empty strings (#7069, @dbczumar)
```

Comment:
This fixes a bug that was (fortunately) never released, so we don't need to call it out. Can we move this to small bug fixes & doc updates?